contents/tutorials/hogql-breakdowns.mdx

```diff
@@ -5,14 +5,32 @@ author: ["ian-vanagas"]
 showTitle: true
 sidebar: Docs
 tags: ['hogql', 'trends', 'product analytics']
----    
+---
+
+import MultiDark from '../images/tutorials/hogql-breakdowns/multi-dark.png'
+import MultiLight from '../images/tutorials/hogql-breakdowns/multi-light.png'
 
 HogQL opens limitless possibilities for how you can breakdown your trends, funnels, and more. This tutorial showcases some of the advanced breakdowns you can create using HogQL.
 
 To create a breakdown using [HogQL](/docs/product-analytics/hogql), create an insight then under "Breakdown by," click "Add breakdown," select HogQL from the options, and add your [expression](/docs/hogql/expressions).
 
 > To understand the full possibilities of HogQL, check out the [available functions in our docs](/docs/product-analytics/hogql#supported-clickhouse-functions) as well as the [events](https://app.posthog.com/data-management/events) and [properties](https://app.posthog.com/data-management/properties) lists from your PostHog instance.
 
+## Multiple properties
+
+HogQL can access multiple properties at once for use in your breakdowns. For example, breakdown by the combination of browser and operating system, you can use the following expression:
+
+```
+concat(properties.$browser, ' ', properties.$os)
```

Comment:
Either of `' / '` or `' — '` would be a better separator then just a space character